src/hotspot/share/opto/vectorIntrinsics.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020, 2023, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2020, 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Sometimes you update ARM copyright, sometimes the Oracle one. Is that intended?

Reply:
Hi, I don't fully understand. I have updated Oracle's copyright year in this file and ARM copyright in the *ad files. Should I not be updating the Oracle's copyright year to 2024?

Reply:
I don't know what is the policy. It is probably ok, so never mind ;)